node/mutations/addUser.ts

```diff
@@ -2,7 +2,7 @@ import { print } from 'graphql'
 import gql from 'graphql-tag'
 
 export default print(gql`
-  mutation(
+  mutation (
```

Comment:
Unfortunately on the lasts PR build, I was forced to execute `eslint --ext js,jsx,ts,tsx . --fix` to pass the checks.